change-notes/1.18/analysis-csharp.md

```diff
@@ -63,4 +56,5 @@
   - `ControlFlowEdgeGotoCase` has been renamed to `ControlFlow::SuccessorTypes::GotoCaseSuccessor`.
   - `ControlFlowEdgeGotoDefault` has been renamed to `ControlFlow::SuccessorTypes::GotoDefaultSuccessor`.
   - `ControlFlowEdgeException` has been renamed to `ControlFlow::SuccessorTypes::ExceptionSuccessor`.
-* The predicate `getCondition()` has been moved from `TypeCase` to `CaseStmt`. It is now possible to get the condition of a `ConstCase` using its `getCondition()` predicate.
+
+> You should update any custom queries that use these entities to ensure that they continue working when the old names are removed in a future release.
```

Comment:
Is the `>` intentional?

Reply:
Thanks for checking. This formatting appears as a note block in the embedded help for LGTM.